examples/creatives/extender.js

```diff
@@ -24,11 +24,12 @@ class Extender extends BaseSimidCreative {
     const params = {
       'duration': mediaDuration + 5,
     };
+
     // Ask the player if we can extend duration 5 seconds to show end cards.
     this.simidProtocol.sendMessage(CreativeMessage.REQUEST_CHANGE_AD_DURATION, params).then(() => {
       this.extendDuration_ = true;
     }).catch(() => {
-      console.log('Player does not support extended duration.');
+      console.log('Player does not support requested duration change.');
```

Comment:
Use sendLog here

Reply:
sendLog is meant to send a message to the Player from the creative. It would not make sense to send a message to the player saying player does not support duration